docs/faq.rst

```diff
@@ -366,6 +366,71 @@ Interestingly, ``jax.grad(divide)(3., 2.)``, works because :func:`jax.grad`
 uses concrete tracers, and resolves the conditional using the concrete
 value of ``y``.
 
+.. _faq-donation:
+
+Buffer donation
+---------------
+
```

Comment:
Should we mention more prominently here (at the top) that buffer donation is not implemented on CPU?

Reply:
Good idea, I added a short sentence at the start.